Description: use private licensecheck module
 For the licensecheck binary package to be Multi-Arch foreign, the perl
 module parts have to be hidden from the world and must thus not be
 installed into the common perl search path. If the module part of
 licensecheck is ever needed by another module, then the module would
 have to be moved to a new binary package and marked as Architecture:any
 Multi-Arch:same to avoid the multiarch interpreter problem.
Author: Johannes Schauer <josch@debian.org>
Last-Update: 2017-02-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/bin/licensecheck
+++ b/bin/licensecheck
@@ -32,6 +32,7 @@
 use Log::Any::Adapter;
 Log::Any::Adapter->set( 'Screen', use_color => $USE_COLOR );
 
+use lib '/usr/share/licensecheck';
 use App::Licensecheck;
 
 =head1 NAME
